test/sass4clj/core_test.clj

```diff
@@ -3,29 +3,8 @@
             [sass4clj.core :refer :all])
   (:import [java.io File]))
 
-(def sass
```

Comment:
Hmm, read the message again. I'm quite sure these tests represent real use case needed by lein/boot tests.